### PR TITLE
{tools}[GCC/11.2.0] Minipolish v0.1.3 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by ItIsI-Orient**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/03f66900415e518e72196dbbde4ce22b for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/m/Minipolish/Minipolish-0.1.3-GCC-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+download_dep_fail = True
+
+options = {'modulename': False}
+
+sanity_check_paths = {
+    'files': ['bin/minipolish'],
+    'dirs': []
+}
+
+moduleclass = "tools"
```
Comment (verdurin) [inline easybuild/easyconfigs/m/Minipolish/Minipolish-0.1.3-GCC-11.2.0.eb:31]:
```suggestion
moduleclass = "bio"
```

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/Minipolish/Minipolish-0.1.3-GCC-11.2.0.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/m/Minipolish/Minipolish-0.1.3-GCC-11.2.0.eb]: Thanks.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3104.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/c96a6830558a458b371bb0e7bcbef689 for a full test report.

**Pull request merged**